benchmark/assert/deepequal-buffer.js

```diff
@@ -1,39 +1,62 @@
 'use strict';
 const common = require('../common.js');
 const assert = require('assert');
+
 const bench = common.createBenchmark(main, {
-  n: [1e3],
+  n: [1e5],
```

Comment:
@mscdex @nodejs/benchmarking Is this sort of benchmark change OK? Do we usually do changes like this, or do we more typically add additional cases (so `n: [1e3, 1e5]`)?